rtrees/src/bktree/tree.rs

```diff
@@ -111,6 +117,7 @@ where
     }
 }
 
+#[allow(unused)]
```

Comment:
Feel free to completely remove this function, now that it is completely replaced.